plugins/org.eclipse.emf.ecoretools.ale.core/src/org/eclipse/emf/ecoretools/ale/core/validation/TypeValidator.java

```diff
@@ -165,6 +167,24 @@ public void setBase(BaseValidator base) {
 			}
 		});
 		
+		
+		
+		clazz.getAttributes().stream()
+			.filter(att -> att.getFeatureRef().getEType() == ImplementationPackage.eINSTANCE.getUnresolvedEClassifier())
```

Comment:
What about extracting the lambda as a method to make the code more expressive?

```java
private List<IValidationMessage> validateBehavioredClass(BehavioredClass clazz) {

    // ...

    clazz.getAttributes().stream()
         .filter(attr -> hasAnUnresolvedType(attr))

    // ...
}

private static boolean hasAnUnresolvedType(Attribute attribute) {
    return attribute.getEType() == ImplementationPackage.eINSTANCE.getUnresolveEClassifier();
}
```

Reply:
I added a small helper method, 
Indeed it will centralized the code when we will have a better unresolve management (ie. a Unresolved instance per searched class /class set)